frontend/apps/os-shell/src/pages/CanonHome.tsx

```
@@ -497,54 +732,576 @@ function CanonContent(): React.ReactElement {
+        keywords: ['network', 'health'],
+        run: runCanonPingCommand,
+      },
+      {
+        id: 'governed-command',
+        title: 'Run Governed Command',
+        subtitle: 'Policy-wrapped command execution',
+        keywords: ['policy', 'governance'],
+        run: runGovernedCommand,
+      },
+    ],
+    [] // eslint-disable-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)